### ISSUE TITLE
Argument for using public key for the path, instead of discovery key

### ISSUE THREAD

**Issue by pfrazee** [dat-ecosystem-archive/dat-secret-storage#3]
The argument for using discovery keys for the folder paths is to make it so you can't enumerate a user's dats with an `ls`.

Counter arguments:

 1. I actually want to be able to do that, especially when I'm debugging.
 2. You're not offering that much protection. If a program can `ls`, it can just read the secret keys. The concern about reading bash history seems esoteric to me.

**Comment by joehand** [comment]
Yep, its a minor protection. The other concern we had was users accidentally publishing keys in screenshots or otherwise.
